src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -269,12 +276,12 @@ public static void SpaceSounds(PlaySoundEvent event) {
         if (event.getSound() != null) {
             if (Minecraft.getInstance().player != null && Minecraft.getInstance().player.level != null && Methods.checkSound(event.getSound().getSource()) && Methods.isSpaceWorldWithoutOxygen(Minecraft.getInstance().player.level)) {
 
-                if (!(event.getSound() instanceof TickableSoundInstance)) {
-                    event.setSound(new SpaceSoundSystem(event.getSound()));
-
-                } else if (event.getSound() instanceof TickableSoundInstance) {
+                if (event.getSound() instanceof TickableSoundInstance) {
```

Comment:
Changed this to an if/else to avoid checking the same thing twice

Reply:
i do that because many other mods does use a own sound system, so i don't brake it

Reply:
Ah okay, I'll undo this part then